model/src/main/scala/hmda/model/institution/Institution.scala

```diff
@@ -0,0 +1,54 @@
+package hmda.model.institution
+
+/**
+ * A financial institution, geared towards requirements for filing HMDA data.
+ */
+case class Institution(
+    id: Int,
```

Comment:
Agreed.  We definitely want to merge this with the existing `Institution`.  We were hesitant to do that as part of this PR while you were out since we didn't have a good understanding of what the implications would be with the Akka Persistence stuff.

I've created https://github.com/cfpb/hmda-platform/issues/472 to get the two implementations merged.